src/verification_store.ts

```
@@ -1,25 +1,49 @@
   private verificationsByDomain: Record<string, DomainVerification> = {};
   private queue: Promise<any> = Promise.resolve();
 
   private constructor() {}
 
+  /**
+   * Enqueues an operation to be executed sequentially
+   * @param operation - Function to be executed in the queue
+   * @returns Promise resolving to the operation result
+   */
   private enqueue<T>(operation: () => Promise<T> | T): Promise<T> {
     return (this.queue = this.queue.then(() => operation()));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)